src/gui/categoryfilterwidget.cpp

```diff
@@ -73,6 +73,7 @@ CategoryFilterWidget::CategoryFilterWidget(QWidget *parent)
     setIconSize(Utils::Misc::smallIconSize());
 #if defined(Q_OS_MAC)
     setAttribute(Qt::WA_MacShowFocusRect, false);
+    setIndentation(0);
```

Comment:
I don't know if it's any different in mac but in KDE if you do this and enable `Use Subcategories` in options, any subcategories vanish from the sidepanel and there doesn't seem to be a way to show them because the expand icon isn't visible.

I realize this is probably out of the scope of this PR, IMO this should be used on all platforms but only when `Use Subcategories` is disabled and maybe reduce the indentation of branches (and the root as much as possible).